app/packages/core/src/components/utils/Options.tsx

```diff
@@ -203,16 +204,20 @@ export const Options: React.FunctionComponent<IOptionsProps> = ({
         <>
           {showSearchButton ? (
             <Button variant="contained" color="primary" size="small" onClick={() => setShow(true)}>
-              {times.time === 'custom'
-                ? `${formatTime(times.timeStart)} to ${formatTime(times.timeEnd)}`
-                : timeOptions[times.time].label}
+              <Typography noWrap={true}>
```

Comment:
without this the selection will wrap in a flex layout where another element tries to get a width of 100%

Reply:
Can you provided an example where this happens please.

Reply:
![image](https://user-images.githubusercontent.com/60616628/222437680-4f2b3dd3-0ef9-4bc5-8d58-614b936617f2.png)

Reply:
Are you using this as follows?

```tsx
<Page
  title="..."
  description="..."
  toolbar={
    <Toolbar>
      <ToolbarItem grow={true}>
        Textfield for search...
      </ToolbarItem>
      <Options ... />
    </Toolbar>
  }
>
  ...
</Page>
```